python/ray/experimental/workflow/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import pytest
+from moto import mock_s3
+from mock_server import *  # noqa
+from ray.experimental.workflow import storage
+
+
+@pytest.fixture(scope="function")
+def filesystem_storage():
+    # TODO: use tmp path once fixed the path issues
+    storage.set_global_storage(
+        storage.create_storage("/tmp/ray/workflow_data/"))
```

Comment:
@suquark there is a bug here about storage.
Basically, if the pash is not /tmp/ray/workflow_data, the recover test will fail.
I wanted to add tests of s3 for recover test for a better coverage (https://gist.github.com/iycheng/aae32fe2c4f73c71a4f2eec357549967), but with s3 it still use the file system based global storage.


Could you help check this issue? Basically, specify the storage explicitly with different paths should work.

After that, I'll add tests for recover path. But this PR is ready for ready which can unblock you.